### ISSUE TITLE
Error: Format not supported + ads

### ISSUE THREAD

**Issue by sqbi87** [nikitakatchik/smarttv-twitch#128]
Hi,

Just recently I started getting this error message while loading stream and randomly during the stream.

When I first load a stream there is an ad playing (ads have never been displaying on TV app before) and after it, an error message appears and I have to reload the stream several times to get it to work. Also as I mentioned the error message can appear randomly during stream requires me to reload.

Is there a way to fix this issue? Help would be much appreciated :)

**Comment by gekyzis** [comment]
Same problem here. Series 6. Fix it please!

**Comment by GurkenKater** [comment]
Same problem here. Not even my pihole could filter this ads.

**Comment by frodebh** [comment]
I have been seeing occasional format not supported for at least the last month, but starting around last week I cant watch any streams, a Amazon Prime ad will start playing and then when the ad is finished the handover to the actual stream will fail. 
When I exit the stream and restart it the same will happen. I am not sure how to reload a stream as mentioned over?

It will be exiting to see if there are still anyone around to fix this thing, hopefully there are!

**Comment by gekyzis** [comment]
As a workaround i use this method posted [here](https://www.reddit.com/r/Twitch/comments/5vaorv/twitch_app_for_samsung_tizen/dgwx06o?utm_source=share&utm_medium=web2x). Hopefully, app will be updated soon :)

**Comment by gekyzis** [comment]
I tried the twitch app yesterday and it worked fine. Same today. Maybe was a problem from twitch's side?
https://ibb.co/vP7TWZX

**Comment by frodebh** [comment]
Twitch is back on their BS. For the last week or so, on opening a stream, an ad for Amazon Prime will run and then when it is supposed to hand over to the selected stream nothing happens. If you're lucky sometimes it will open the stream directly without the ad and it works. Same as last time.
I have ordered a €30 Android TV box from China for the single purpose of watching Twitch on my beloved 2014 Samsung TV. A big thanks go out to the devs for all the entertainment this app provided ✨

**Comment by henriquelbsouza** [comment]
Yeah happens to me aswell, they are fucking up when switching from stream -> ad -> stream back aswell...
In some streams it even says: Twich AD ~something~...

It just says format not supported, maybe it is fixable in the app, not sure.

**Comment by frodebh** [comment]
I don't get format not supported here, only a black screen. Maybe this current issue can be fixed but I'm not optimistic about unofficial apps in the long run unfortunately ..

**Comment by sqbi87** [comment]
Hi,
The problem with **error: format not supported** got worst. Now I can't open any stream unless I manually type the channel name.
Is there any possibility to fix this problem?
@nikitakatchik or anyone? :)
